### PR TITLE
Adding YAML Test Cases - Nov 12th

### PR DESCRIPTION
**Pull request by ghost**
Test Script Added
Adding YAML test scripts for
1.Test_TC_TSTAT_2_2
2.Test_TC_WNCV_2_4
3.Test_TC_DIAGSW_2_1
Generated script files

Testing
Executed all YAML test scripts, attached logs for reference
[allrun.txt](https://github.com/project-chip/connectedhomeip/files/7527664/allrun.txt)

### CODE REVIEW

**Comment by bzbarsky-apple** [comment]
@manjunathambiger23 please note the CLA bits?

**Comment by woody-apple** [comment]
Fast tracking, given this is adding tests.

**Comment by manjunathambiger23** [comment]
rebased to solve merge conflicts.

File: examples/chip-tool/templates/tests.js [hunk 1]
```diff
@@ -148,13 +149,14 @@ function getTests()
...
   const ThreadNetworkDiagnostics = [
     'Test_TC_DIAGTH_1_1',
-    'Test_TC_DIAGSW_3_2',
   ];
 
   const WindowCovering = [
     'Test_TC_WNCV_1_1',
     'Test_TC_WNCV_2_1',
     'Test_TC_WNCV_2_2',
+    'Test_TC_WNCV_2_3',
```
Comment (jmeg-sfy) [inline examples/chip-tool/templates/tests.js:158]: Please remove 2.3 , i ll take care of it , it is far more complex and the WindowCovering is nor ready nor spec compliant

File: src/app/tests/suites/certification/Test_TC_WNCV_2_3.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+tests:
+    - label: "sends UpOrOpen command to DUT"
+      command: "UpOrOpen"
+
+    #Issue #10811 if condition is required, need YAML support for this
+    - label: "Reads OperationalStatus"
+      command: "readAttribute"
+      attribute: "Type"
+      response:
+          constraints:
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_2_3.yaml:30]: Please remove this file from the testing

File: src/app/tests/suites/certification/Test_TC_WNCV_2_4.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+tests:
+    #Issue #10811 if condition is required, need YAML support for this
+    - label: "Reads Type attribute from DUT"
+      command: "readAttribute"
+      attribute: "Type"
+      response:
+          value: 0
+
+    - label: "Reads Type attribute constraints"
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_2_4.yaml:29]: This is one is ok thank you for the work

File: src/darwin/Framework/CHIP/templates/tests.js [hunk 4]
```diff
@@ -153,6 +154,8 @@ function getTests()
     'Test_TC_WNCV_1_1',
     'Test_TC_WNCV_2_1',
     'Test_TC_WNCV_2_2',
+    'Test_TC_WNCV_2_3',
```
Comment (jmeg-sfy) [inline src/darwin/Framework/CHIP/templates/tests.js:157]: Please remove 2.3
```suggestion
```

**Review by jmeg-sfy — CHANGES_REQUESTED** [review]

File: src/darwin/Framework/CHIP/templates/tests.js (hunk 4 above)
Comment (manjunathambiger23) [reply src/darwin/Framework/CHIP/templates/tests.js]: Removed Testcase TC-WNCV-2.3

**Review by jmeg-sfy — APPROVED** [review]

**Comment by manjunathambiger23** [comment]
> @manjunathambiger23 please note the CLA bits?

I have signed in the CLA agreement but it is still not refelcting in the PR
image

**Comment by andy31415** [comment]
@manjunathambiger23 - you make have commited as a different user than you are logged in to github - I had this problem when using 2 separate accounts. Please sign the CLA with all the users that contributed

**Comment by andy31415** [comment]
> @manjunathambiger23 - you make have commited as a different user than you are logged in to github - I had this problem when using 2 separate accounts. Please sign the CLA with all the users that contributed

Actually log seems to say "“manjunathambiger23” seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account" ... I do not quite understand why. However we require CLA for any merges.

**Comment by manjunathambiger23** [comment]
> > @manjunathambiger23 - you make have commited as a different user than you are logged in to github - I had this problem when using 2 separate accounts. Please sign the CLA with all the users that contributed
> 
> Actually log seems to say "“manjunathambiger23” seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account" ... I do not quite understand why. However we require CLA for any merges.

I will close this PR and try it again with CLI agreed git account

**Comment by manjunathambiger23** [comment]
I will close this PR and try it again with CLI agreed git account

**Pull request closed without merging**